cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -2097,10 +2097,11 @@
   </property>
 
   <property>
-    <name>metrics.v2.table.scan.enabled</name>
-    <value>false</value>
+    <name>metrics.v3.table.splits</name>
+    <value>16</value>
     <description>
-      Enable or disable scanning v2 metrics tables. By default set to false.
+      Number of splits for all v3 metrics tables. This property can only be changed when v3 tables are created for the
```

Comment:
Also, users do not know when the tables are created for the first time. Perhaps say: can only be changed before CDAP starts up for the first time and creates the metrics tables.